### ISSUE TITLE
Bounding Box location

### ISSUE THREAD

**Issue by 2akash111998** [michhar/pytorch-yolo-v3-custom#7]
I'm getting this error:
  File "train.py", line 238, in <module>
    for image, ground_truth in tqdm(data_loader):
  File "/home/ajinkya/.local/lib/python3.6/site-packages/tqdm/std.py", line 1097, in __iter__
    for obj in iterable:
  File "/home/ajinkya/.local/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 346, in __next__
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/ajinkya/.local/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/ajinkya/.local/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/media/ajinkya/common/object_locate/pytorch-yolo-v3-custom/customloader.py", line 287, in __getitem__
    _img, ground_truth = self.det_transforms(_img, ground_truth)
  File "/media/ajinkya/common/object_locate/pytorch-yolo-v3-custom/data_aug/data_aug.py", line 58, in __call__
    images, bboxes = augmentation(images, bboxes)
  File "/media/ajinkya/common/object_locate/pytorch-yolo-v3-custom/data_aug/data_aug.py", line 997, in __call__
    if bboxes.size > 0:
AttributeError: 'NoneType' object has no attribute 'size'

I feel this has something to do with the annotation file location. I have labelled my dataset using labelme and converted the json to yolo txt format.
My dataset location is /data/obj/
and the annotations are stored in the same location with same name as image but in txt format.

**Comment by michhar** [comment]
Correct, this project does not support, yet, background image addition.  This is a good suggestion.

**Comment by 2akash111998** [comment]
> Correct, this project does not support, yet, background image addition. This is a good suggestion.
> 
> You could, as workaround (see [this issue](https://github.com/pjreddie/darknet/issues/898)) try for a background image with a new label (say you have two classes, 0 and 1, so BG could be 2), but I haven't tried this. Let me know your thoughts on this approach. Thanks.
> 
> ```
> 2 0.0 0.0 0.0 0.0
> ```

Well this makes sense but I encounter another issue as follows
![Screenshot from 2019-12-08 20-11-14](https://user-images.githubusercontent.com/35839624/70391056-f0c03900-19f6-11ea-8d9e-442544e085f7.png)
